p2panda-rs/src/document/document_view_id.rs

```diff
@@ -63,6 +64,17 @@ impl DocumentViewId {
     }
 }
 
+impl Display for DocumentViewId {
+    /// Document view ids are displayed by concatenating their hashes with an underscore separator.
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        for (i, operation_id) in self.0.clone().into_iter().enumerate() {
+            let separator = if i == 0 { "" } else { "_" };
```

Comment:
Check out `peek()` which helps with this kind of pattern : https://doc.rust-lang.org/std/iter/struct.Peekable.html

Not strictly necessary here cos it's already very neat, but just for interest.

Reply:
I saw that you used that yesterday hehe, also quite interesting.